rero_ils/modules/patron_transactions/jsonschemas/patron_transactions/patron_transaction-v0.0.1.json

```diff
@@ -0,0 +1,116 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "title": "Patron Transaction",
+  "description": "JSON schema for Patron Transaction.",
+  "additionalProperties": false,
+  "required": [
+    "$schema",
+    "pid"
+  ],
```

Comment:
Can you add fields `creation_date`, `status`, `type`, `patron` as required field for this resource.

If you describe a PatronTransaction with `type==overdue`, the `loan` fields is also required (not for other type).
You can do that using `oneOf` directive.